projects/rawkode-academy/web/src/components/partner/pricing/featured.vue

```diff
@@ -8,7 +8,7 @@ interface PricingPlan {
 
 const maltPricingPlan: PricingPlan = {
 	title: "Malt",
-	price: 96000,
+	price: 96,000,
```

Comment:
Because this field is typed as an int, you may need to change them to strings for the formatting with the comma.

Reply:
ah, touche

Reply:
also shame on me for not running the site locally to test this :crying_cat_face:

Reply:
all fixed